five reminders displaying on the dom from mirage data, Closes #1

noahpeden opened the pull request:
## Purpose

Populate page with five fake reminders from mirage data, and have class name of .spec-reminder-item for each reminder. 
Issue 1: https://github.com/turingschool-projects/1610-remember-8/issues/1


## Approach

Populates page with five fake reminders from mirage data.
We also took the approach of generating a reminders route, then overwriting the home route with index.

### Learning


We both did tutorials for a day before this stage including youtube videos and the ember tutorial on yoember.com.


### Test coverage 

We didn't write the tests, but it passes the already written test of having five reminders display on the DOM.



### Follow-up tasks

- [Issue #2 ](https://github.com/turingschool-projects/1610-remember-8/issues/2)

### Screenshots


#### Before
<img width="1440" alt="screen shot 2017-02-08 at 4 01 53 pm" src="https://cloud.githubusercontent.com/assets/15061527/22761455/0042923c-ee18-11e6-889b-f9c5fb8ff164.png">

#### After
<img width="1438" alt="screen shot 2017-02-08 at 4 00 57 pm" src="https://cloud.githubusercontent.com/assets/15061527/22761466/09020448-ee18-11e6-84a7-00cebe38fe82.png">

noahpeden committed:
five reminders displaying on the dom from mirage data, closes #1

martensonbj commented on app/routes/index.js:
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
```
Clean up extra spaces

martensonbj commented on app/routes/reminders.js:
```diff
@@ -0,0 +1,7 @@
+
+export default Ember.Route.extend({
+  model(){
+    return [1, 2, 3, 4, 5]
```
Because we're using Mirage we can make a query to the store to findAll records. Check out [the docs](https://guides.emberjs.com/v2.11.0/models/finding-records/#toc_retrieving-multiple-records).

martensonbj commented on app/templates/index.hbs:
```diff
@@ -0,0 +1,7 @@
+{{#each model as |reminder|}}
```
This should live in the `reminders` template since it displays all reminders. We actually don't need this file at all.

martensonbj commented on app/templates/reminders.hbs:
```diff
@@ -0,0 +1,3 @@
+{{#each model as |reminder|}}
```
See comment in index.hbs

noahpeden replied:
Hi! We fixed this in the issue #2 commit, but I changed it here as well

martensonbj commented on tests/unit/models/index-test.js:
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
```
We dont need this test or file

martensonbj commented on tests/unit/routes/index-test.js:
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Also don't need this file

martensonbj commented on tests/unit/routes/reminders-test.js:
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Routes get tested by default in acceptance tests with lines like `visit('/')` or `currentURL('/')` etc. We dont need this file.

martensonbj reviewed (changes requested):
Good start team! Few changes requested.

noahpeden committed:
Update index.js

noahpeden committed:
Update reminders.js

noahpeden committed:
Update index.hbs

@martensonbj this file is deleted in the branch for issue #2, but I've deleted it here as well

noahpeden committed:
Update reminders.hbs

noahpeden committed:
Update index-test.js

noahpeden committed:
Update index-test.js

noahpeden committed:
Update reminders-test.js

For all the files you've asked to delete, I trashed them in Atom, but here I've just deleted all the code. In the next issue we'll make all these changes in Atom, then make one small, atomic commit.

noahpeden reviewed (commented):
Hi Brenna! We've committed all the changes you asked to make, but we can also make another commit with the trashed files if you want.  Also we had already made most of these changes for the issue #2 branch that justin pushed up yesterday.

martensonbj commented on app/routes/index.js:
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
```
We don't need this anymore since the logic is in the reminders route - this file should be used to redirect to `'/reminders'`

martensonbj commented on app/templates/index.hbs:
```diff
@@ -0,0 +1 @@
+
```
This can be deleted

martensonbj commented on tests/unit/models/index-test.js:
```diff
@@ -0,0 +1 @@
+
```
We won't need this file

martensonbj commented on tests/unit/routes/index-test.js:
```diff
@@ -0,0 +1 @@
+
```
Dont' need this file either

martensonbj commented on tests/unit/routes/reminders-test.js:
```diff
@@ -0,0 +1 @@
+
```
For the most part route tests happen organically in acceptance tests we can delete this file

martensonbj reviewed (changes requested):
Almost there - we have some repetition that should cause your tests to be failing. Check out the comments

noahpeden committed:
fixed requests from Brenna about issue #1, mostly deleting files

noahpeden committed:
fixed Brenna's changes, mostly deleting files for issue #1

noahpeden reviewed (commented):
@martensonbj this latest commit should reflect all the changes you requested, mostly deleting those file
